Sprint-1/interpret/to-pounds.js

```diff
@@ -25,3 +26,16 @@ console.log(`£${pounds}.${pence}`);
 
 // To begin, we can start with
 // 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 2.const penceStringWithoutTrailingP = penceString.substring(0, penceString.length - 1);
+//    The substring() method her returns the part of this string from the start index up to lenght-1 => last one not inclouded then
+//    initialises a string variable  assign to the  result of supStr.
+// 3.const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+//   initialises a string variable with value of the result of padStart this case it will
+//   retun the value of three index form the started of Str =>399.
+// 4.const pounds = paddedPenceNumberString.substring( 0,paddedPenceNumberString.length - 2);
+//  samilar to 2 just in this case last to index not inclouded
+// 5.const pence = paddedPenceNumberString.substring(paddedPenceNumberString.length - 2).padEnd(2, "0");
+// this will finsh i 4 steps A/initialises a string variable/B- git the value (paddedPenceNumberString.length - 2 == 1)and  take us arg
+//  C/ calc paddedPenceNumberString.substring(1)=="99"  and the resultof padEnd() = 99 asign tp the pence
```

Comment:
Can we expect this program to work as intended even if we deleted `.padEnd(2, "0")` from the code?